source/release_notes.rst

```diff
@@ -4,6 +4,91 @@ Release Notes
 #############
 
 .. contents:: :local:
+---------------
```

Comment:
@swaroopbalan your have to add one blank line after previous directive.

Reply:
done